gigya_raas/gigya_raas.install

```diff
@@ -27,34 +27,49 @@ function gigya_raas_install() {
 	];
 
 	$schema = Database::getConnection()->schema();
+	$logger = \Drupal::logger('gigya_raas');
 
-	$error_message_format = sprintf('Unable to create column `%s` in the `%s` table during Gigya RaaS module installation.
-														If the column already exists from a previous installation of the module, you may ignore this warning. If the column already exists from another module, please contact Gigya\'s support.
-														If the column does not exist, please try to reinstall the module.');
+	$error_message_format = 'Unable to create column `%column` in the `%table` table during Gigya RaaS module installation. If the column already exists from a previous installation of the module, you may ignore this warning. If the column already exists from another module, please contact Gigya\'s support. If the column does not exist, please try to reinstall the module.';
 	/* Create two new columns. The reason they are separated is that if one exists from a previous install, it should be able to create both, not fail */
 	try {
 		$schema->addField('sessions', 'expiration', $spec_expiration);
 	} catch (Exception $e) {
-		\Drupal::logger('gigya_raas')->warning(sprintf($error_message_format, 'expiration', 'sessions'));
+		$logger->warning($error_message_format, [
+			'%column' => 'expiration',
+			'%table' => 'sessions',
+		]);
 	}
 	try {
 		$schema->addField('sessions', 'is_remember_me', $spec_is_remember_me);
 	} catch (Exception $e) {
-		\Drupal::logger('gigya_raas')->warning(sprintf($error_message_format, 'is_remember_me', 'sessions'));
+		$logger->warning($error_message_format, [
+			'%column' => 'is_remember_me',
+			'%table' => 'sessions'
+		]);
 	}
 
 	$current_user = \Drupal::currentUser();
 	$admin_id = $current_user->id();
 
 	/* Make sure the administrator doing the installation has the proper session set */
 	if (in_array('administrator', $current_user->getRoles())) {
-		Database::getConnection()->query('UPDATE {sessions} s SET expiration = :expiration WHERE s.uid = :uid', [
+		Database::getConnection()->query('UPDATE {sessions} SET expiration = :expiration WHERE s.uid = :uid', [
```

Comment:
Seems like this will break. Can you please remove this part and target develop? (already addressed this anyway in another PR). Thanks.

Reply:
It is fixed in other place https://github.com/gigya/drupal8/pull/49/files